clang/test/CheckedCRewriter/typedefs.c

```diff
@@ -0,0 +1,41 @@
+// RUN: cconv-standalone -alltypes -addcr %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
```

Comment:
I would love if it for you to add typedefs in a few more places in your test, e.g., as return types from functions, parameters to functions, part of struct typedefs (e.g., `typedef struct foo { int *x; } *foo_ptr;` -- just want to make sure your rewriting code is covering the needed places.

Reply:
I think those are covered now, correct me if I'm missing something

Reply:
BTW, this shouldn't be `cconv-standalone` anymore, right? Should be `3c` everywhere.

Reply:
Yes just pushed on that.

Reply:
This file is dead; this directory should not exist.